packages/components/icon/src/icon.vue

```diff
@@ -28,7 +28,7 @@ const fontSize = computed(() => {
 const style = computed(() => {
   return {
     fontSize: fontSize.value,
-    color: props.color,
+    color: props.isDisabled ? '#BBBBBB' : props.color,
```

Comment:
should you not use the TW defined colour instead ?

Reply:
Hehehe done 😎